src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -162,17 +162,23 @@ public function get( WP_REST_Request $request ) {
 			$data['rest_url'] = $this->get_current_rest_url( $args, $extra_rest_args );
 
 			$events = tribe_events();
+			$tmp = array();
 
-			if ( ! empty( $request['runs_between'] ) && $args['start_date'] && $args['end_date'] ){
-				$events = $events->where( 'runs_between', $args['start_date'], $args['end_date'] );
+			if ( ! empty( $request['runs_between'] ) ){
+				$events = $events->by( 'runs_between', ...$request['runs_between'] );
```

Comment:
Is this line PHP +5.6 compatible?

Reply:
Should be - it's what's being used here: https://github.com/moderntribe/the-events-calendar/pull/2548/files#diff-11b3f3a370885fc38177fe5c2798ce9fR1410